#include "IntergrationData.h"

SignalInfoStruct			signalInfo;
MsgDataStruct				msgData;
vector<SignalInfoStruct>	signalInfo_Vector;
vector<MsgDataStruct>		msgData_Vector;
TestCaseStruct				testCase_Struct;
vector<TestCaseStruct>		testCase_Vector;
vector<string>				case_Level;
vector<int>					case_LevelFlag;
vector<SignalInfoStruct>	case_TestStep;
vector<int>					case_TestValue;
vector<SignalInfoStruct>	case_DesireSignalName;
vector<int>					case_DesireValue;


MsgProcess::MsgProcess()
{
	signalInfo.msg_Name = "";
	signalInfo.id = 0;
	signalInfo.signal_Name = "";
	signalInfo.signal_Length = 0;
	signalInfo.signal_StartBit = 0;
	signalInfo.signal_StartByte = 0;
	signalInfo.signal_Factor = 0;
	signalInfo.signal_Offset = 0;

	msgData.msg_Name = "";
	msgData.id = 0;
	for (int i = 0; i < 8; i++)
	{
		msgData.data[i] = 0;
	}
	msgData.msg_Checksum = 0;
	msgData.msg_LiveCouter = 0;
}

MsgProcess::~MsgProcess()
{

}

void MsgProcess::FillMsgData(){
	testCase_Struct.test_Function_Name = "A001";
	testCase_Struct.test_Function_Name.push_back("A001-001");
	testCase_Struct.case_LevelFlag.push_back(0);
	testCase_Struct.case_Level.push_back("A");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("1");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("-");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.test_Function_Name.push_back("A001-002");
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("A");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("1");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("-");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.test_Function_Name = "A002";
	testCase_Struct.test_Function_Name.push_back("A002-001");
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("A");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("2");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("-");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.test_Function_Name.push_back("A002-002");
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("A");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("0");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("2");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	testCase_Struct.case_LevelFlag.push_back(0);
	ttestCase_Struct.case_Level.push_back("-");
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "AvailableSt_EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "VehiclSpeed_Q";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "DrivingDirection";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_1";
	signalInfo.signal_Name = "EPBSwitch";
	signalInfo.id = 0xc0;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_FR";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RL";
	signalInfo.id = 0x130;
	signalInfo.msg_Name = "ABS_3";
	signalInfo.signal_Name = "WheelSpeedDirection_RR";
	signalInfo.id = 0x130;
	

}